src/web-components/img-adv/img-adv.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Created by imamudin.naseem on 02/09/16.
+ */
+
+'use strict'
+import h from 'snabbdom/h'
+import R from 'ramda'
+
+const init = () => ({})
```

Comment:
There is no initial state required. We are not doing any internal calculation that requires initial state